File: java/Problem1160.java
import java.util.Scanner;

public class Problem1160 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int t = sc.nextInt();

        for (int i = 0; i < t; i++) {
            int pa = sc.nextInt();
            int pb = sc.nextInt();

            double g1 = sc.nextDouble() / 100.0;
            double g2 = sc.nextDouble() / 100.0;
            int anos = 0;
            while (pa <= pb) {
                pa += pa * g1;
                pb += pb * g2;
                anos++;
                if (anos > 100) {   
                    break;
                }
            }

            if (anos > 100) {
                System.out.println("Mais de 1 seculo.");
            } else {
                System.out.printf("%d anos.\n", anos);
            }
        }

        sc.close();
    }
}

File: python/problem1160.py
n = int(input())
I = 0

while I < n:
        
    pa, pb, ga, gb = map(float, input().split())
    ga /= 100
    gb /= 100
    II = 0
    while II <= 100:
        if pa > pb:
            break
        pa = int(pa * ( 1 + ga))
        pb = int(pb * ( 1 + gb))
        
        II += 1
    if II > 100:
        print('Mais de 1 seculo.')
    else:
        print('%d anos.' % II)
    I += 1
